src/pages/selectAppointment.tsx

```diff
@@ -36,28 +31,38 @@ export default function Home() {
       });
   }, [supabaseClient, userStatus.user]);
 
-  if (userStatus.isLoading || !appointments) return <Loading />;
+  if (userStatus.isLoading) return <Loading />;
   if (!userStatus.user) {
     router.push(PAGE_PATHS.LOGIN);
     return;
   }
+  if (!userStatus.user.isVerified) {
+    router.push(PAGE_PATHS.HOME);
+    return;
+  }
+  if (appointments == null) return <Loading />;
   return (
     <>
       <Navbar />
-      <Typography paddingTop="40px" variant="h4" align="center">
-        Select Appointment To Confirm
-      </Typography>
-      <Box display="flex" justifyContent="center" padding="40px">
-        <Grid container spacing="40px" width="80vw">
-          {appointments.map((item, index) => (
-            <Grid item xs={12} md={6} key={index}>
-              <div onClick={() => handleCardClick(item.appointmentId)}>
-                <AppointmentCard appointment={item} />
-              </div>
+
+      <Stack spacing={4} alignItems="center" style={{ marginBottom: "3vh" }}>
+        {/* Page header */}
+        <Box sx={{ marginTop: "3vh" }}>
+          <Typography variant="h1">Select Appointment To Confirm</Typography>
+        </Box>
+        <Grid
+          container
+          justifyContent="space-between"
+          rowSpacing={6}
+          style={{ width: "80vw", minWidth: "1050px", marginTop: -6 }}
+        >
+          {appointments.map((appointment, index) => (
+            <Grid item key={index} xs={5.75}>
+              <AppointmentCard appointment={appointment} prefix={PAGE_PATHS.CONFIRM_APPOINTMENT} />
```

Comment:
User own appointment should not appear on this page. (I think this is the question @Jinnapat asked on discord so it might be backend problem)

Reply:
@Jinnapat @chanathip101 Is one of you guys available for this?

Reply:
On it

Reply:
Should be fixed now